common/utils.py

```diff
@@ -48,3 +50,10 @@ def get_date(day_arg):
     """
     day_num = _get_day_arg(day_arg)
     return date.today() + timedelta(days=day_num)
+
+
+def render(filename, context={}):
+    path = 'templates'
+    return jinja2.Environment(
+        loader=jinja2.FileSystemLoader(path)
```

Comment:
nitpick: Would the value for `path` ever be different ? If not, might be cleaner to just pass `templates` into the `FileSystemLoader` function